test/unit/plugins/guests/linux/cap/change_host_name_test.rb

```diff
@@ -0,0 +1,77 @@
+require_relative "../../../../base"
+
+describe "VagrantPlugins::GuestLinux::Cap::ChangeHostName" do
+  let(:described_class) do
```

Comment:
This should be grabbing the capability directly from the plugin, not creating a new class to extend the module into:

```ruby
let(:described_class) do
  VagrantPlugins::GuestLinux::Plugin.
    components.guest_capabilities[:linux].get(:change_host_name)
end
```